### PR TITLE
content profiles (SD-3871)

### PR DESCRIPTION
**Pull request by gbbr**
**Demo**:
See [video](https://youtu.be/pKhN8vYiAgs) for details.

**Fixes issues**:
[SD-3871](https://dev.sourcefabric.org/browse/SD-3871) - Create a content profile
[SD-3894](https://dev.sourcefabric.org/browse/SD-3894) - Edit a content profile

It also fixes the front-end part of [SD-3895](https://dev.sourcefabric.org/browse/SD-3895) (Delete a content profile) and part of [SD-3896](https://dev.sourcefabric.org/browse/SD-3896) (Manage content profile fields).

### CODE REVIEW

File: scripts/superdesk-workspace/content/views/profile-settings.html [hunk 1]
```diff
@@ -1,40 +1,103 @@
 <div sd-settings-view>
-    <div class="split-content">
-        <div class="header">
-            <h2 translate>Content Profiles</h2>
+	<div class="split-content">
```
Comment (vied12) [inline scripts/superdesk-workspace/content/views/profile-settings.html]: we used to indent with spaces

**Comment by ahilles107** [comment]
I like PR's with video! One question - why we can't edit profile properties when we create new one (another click on edit is require)? It's not so intuitive for user.

File: scripts/superdesk-workspace/content/views/schema-editor.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<div class="schema-editor">
```
Comment (vied12) [inline scripts/superdesk-workspace/content/views/schema-editor.html]: Same here, it would be better to use spaces for indentation

**Comment by gbbr** [comment]
@ahilles107 Yes, this was the same point brought up by @fritzSF - we would have to do this as a follow up and re-think the specs and workflow a bit. Currently the user must first create a profile before he is able to edit the fields. This is because (according to the issue description) when a user wants to create a new profile he should specify _Title_ and _Description_ only. Afterwards, the server decides what fields to pre-populate the profile with as a default, which are then returned to the client. Basically the client is not aware of the fields on a content profile before it is created.

One suggestion @fritzSF had was to replace the "Save" button on the creation form with a "Next", which will automatically open the editing form after creation and providing a better user experience which is closer to yours and his suggestion.

I hope this clarifies your question a bit. I recommend that we do this as a follow up though, because it is a bit out of scope for this sprint.

**Comment by gbbr** [comment]
@vied12 addressed

File: scripts/superdesk-workspace/content/views/schema-editor.html (hunk 2 above)
Comment (gbbr) [inline scripts/superdesk-workspace/content/views/schema-editor.html]: I replaced all tabs with 4 spaces.

**Comment by ahilles107** [comment]
I totally agree with @fritzSF idea about "next" button. It would solve that issue. Follow up sounds good - but please make sure that it will not end in backlog and live there till end of jiira.

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**